rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :rentals
+  attr_reader :person
```

Comment:
- [ ] As mentioned by the previous reviewer, there needs to be a `Getters and Setters for @book and @person`. currently, `person`  only has a `getter`. I suggest you update this as required. :+1: :100: Here's what your code could look like. :+1: 

```ruby
attr_accessor :date, :book, :person
```